### ISSUE TITLE
How to unlock Protected Achievements?

### ISSUE THREAD

**Issue by ET001** [gibbed/SteamAchievementManager#176]
Some achievements cannot be unlocked and its showing that it is a protected achievement, and marked as 'red'. 

![image](https://user-images.githubusercontent.com/71649829/93773873-ec060200-fc63-11ea-8f09-0ff1d9609ded.png)

Is there any way to unlock them?

**Comment by alexmehta** [comment]
As you can see by the message, it cannot be managed with Steam Achievement Manager, meaning you cannot unlock them using the manager.

**Comment by ScienceDiscoverer** [comment]
But is it any way it could potentially be unlocked, even theoretically? I have a game that no longer exists in Steam, and i have foolishly gotten 1 achievement in it, and now it messes up my total achievement %...

**Comment by alexmehta** [comment]
You can unlock it by doing what the achievement wants you to do. The achievements are locked because they are server side achievements.

**Comment by ScienceDiscoverer** [comment]
I guess only way out here is to battle steam support endlessly and relentlessly so they would provide a way to COMPLETELY remove a game from one's account, with achievements rest and stats and everything else. This is just absurd that we don't have ability like this still....

**Comment by alexmehta** [comment]
Yes. @ScienceDiscoverer I'm not sure what you are expecting from steam achievement manager. The manager uses files in steam in order to mark achievements as done. It can't manage something that's not locally on your machine.

**Comment by ScienceDiscoverer** [comment]
@alexmehta I was not expecting this from SAM, I am expecting this from STEAM devs. Cos for now SAM is only thing that gives us at least some control.

**Comment by ToadCast** [comment]
Then I understand you question has been answered, please consider closing this issue. I hope you found some help on Steam's side.

**Comment by kerryemerson** [comment]
I came across this issue when trying to 'achieve' the final achievements I didn't get for Far Cry 5, due to the fact that the multiplayer and arcade modes crash now, as presumably the servers are no longer up and running. As the game launches through Ubisoft launcher (via Steam). Shame I can't get 100% completion, through no fault of my own. :(
